src/tests/CompleteOptions.test.tsx

```diff
@@ -0,0 +1,71 @@
+import '@testing-library/jest-dom';
+
+import { h } from 'preact';
+import { clickPurposeCheckbox, getPurposeCheckState, render } from './utils';
+import { test, expect } from '@jest/globals';
+
+import { CompleteOptions } from '../components/CompleteOptions';
+import { fireEvent } from '@testing-library/preact';
+import { MOCK_PURPOSES } from './constants';
+import { DEFAULT_PURPOSE_TO_MESSAGE_KEY } from '../components/constants';
+import { sortByPurposeOrder } from '../helpers';
+
+test('CompleteOptions', () => {
```

Comment:
I'd recommend that we structure these tests similarly to the rest of our codebase, with a top level `describe` with individual `test` underneath that for each test case. That makes the tests easier to understand/parse and allows us to reuse the setup constants across multiple test runs if needed